src/signer/tests/it/utils/pocketic.rs

```diff
@@ -82,20 +96,36 @@ impl BackendBuilder {
     /// To override, please use `with_cycles()`.
     pub const DEFAULT_CYCLES: u128 = 2_000_000_000_000;
     /// The default Wasm file to deploy:
-    /// - If the environment variable `BACKEND_WASM_PATH` is set, it will use that path.
-    /// - Otherwise, it will use the `BACKEND_WASM` constant.
+    /// - If the environment variable `SIGNER_CANISTER_WASM_FILE` is set, it will use that path.
```

Comment:
Is this the env var set by dfx?  I can't check the docs right now as this is down: https://internetcomputer.org/docs/current/developer-docs/developer-tools/cli-tools/cli-reference/dfx-envars

Reply:
No, this is set in the `test.backend.sh`. The test doesn't use `dfx`, so not sure about relying on it.